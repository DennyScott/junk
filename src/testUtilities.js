import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

export const rendersCorrectly =  component => {
  const wrapper = shallow(component);
  expect(wrapper.length).toBe(1);
}

export const matchesSnapshot = component => {
  const wrapper = shallow(component);
  expect(toJson(wrapper)).toMatchSnapshot();
}