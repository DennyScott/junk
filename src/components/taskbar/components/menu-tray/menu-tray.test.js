import React from 'react';
import MenuTray from './menu-tray';
import { rendersCorrectly, matchesSnapshot } from 'testUtilities';

const minProps = {};

const getMinComponent = () => <MenuTray {...minProps} />;

it('MenuTray renders correctly', () => {
  rendersCorrectly(getMinComponent(), expect);
});

it('MenuTray render matches snapshot', () => {
  Date.now = jest.fn(() => new Date(Date.UTC(2017, 7, 9, 8)).valueOf());
  matchesSnapshot(getMinComponent(), expect);
});
