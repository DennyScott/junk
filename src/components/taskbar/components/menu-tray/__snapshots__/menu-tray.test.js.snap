// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuTray render matches snapshot 1`] = `
<styled.div>
  <styled.div>
    10:26 PM
  </styled.div>
</styled.div>
`;
